atd-vze/src/views/Crashes/Recommendations.js

```diff
@@ -1,61 +0,0 @@
-import React, { useState } from "react";
```

Comment:
This file was moved to a new folder before I touched this code for organization which I totally support ❤️